qwik/lib/src/components/logo/logo.tsx

```diff
@@ -0,0 +1,16 @@
+import { component$ } from '@builder.io/qwik';
+
+export const Logo = component$(() => {
+  return (
+    <div>
+      <a href="https://qwik.dev/">
+        <img
+          alt="Qwik Logo"
+          width={400}
+          height={147}
+          src="https://cdn.builder.io/api/v1/image/assets%2FYJIGb4i01jvw0SRdL5Bt%2F667ab6c2283d4c4d878fb9083aacc10f"
```

Comment:
seems like the (very) old logo of Qwik.
I guess this starter has been generated from an old version of Qwik?

Reply:
This is the logo in the library template at the moment: https://github.com/QwikDev/qwik/blob/main/starters/apps/library/src/components/logo/logo.tsx

I can replace it with what you are using on the docs or something else if you prefer? (the docs are not using an SVG exactly, but I think I can make it work). Also do you need this change in the Qwik repo as well?
The docs logo I could find: https://github.com/QwikDev/qwik/blob/main/packages/docs/src/components/svgs/qwik-logo.tsx

Reply:
@Mivr yeah good point, we need to update it, we'll do